src/areaSearch/reducer.js

```diff
@@ -13,6 +13,8 @@ import type {
 import type {ReceiveAttributesAction, ReceiveMethodsAction} from '$src/areaSearch/types';
 import type {ApiResponse} from '$src/types';
 import merge from 'lodash/merge';
+import type { Set } from "immutable";
```

Comment:
This also has `' and "` plus `{ thing } and {thing}`

Reply:
This was also an unused module, so I removed it.